src/index.html

```diff
@@ -10,7 +10,48 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+  
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="#" 
+      method="get" 
+      data-qa="big"
+      class="form"
+    >
+      <img 
+        src="./images/Search.svg" 
+        alt="search" 
+        class="search_svg"
+      >
+      
+      <input 
+        class="search_bar"
+        type="search" 
+        name="search_bar" 
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
+
+    <form 
+      action="#" 
+      method="get" 
+      data-qa="small"
+      class="smallForm"
+    >
+
+      <img 
```

Comment:
Don't add spaces between a parent and a child
```suggestion
    >
      <img 
```